plugins/repository-hdfs/src/test/resources/rest-api-spec/test/hdfs_repository/20_repository.yaml

```diff
@@ -0,0 +1,25 @@
+# Integration tests for HDFS Repository plugin
+#
+# Check plugin is installed
+#
+"HDFS Repository Config":
+    - do:
+        snapshot.create_repository:
+          repository: test_repo_hdfs_1
+          verify: false
+          body:
+            type: hdfs
+            settings:
+              # local HDFS implementation
+              conf.fs.es-hdfs.impl: "org.elasticsearch.repositories.hdfs.TestingFs"
```

Comment:
This seems like it defeats the purpose of an integration test. ES should be running as real as it would be outside of testing. I understand that we need an easy way to start an associated process with the integration tests (I am working on that), but I don't think we should do this hack, I'm not sure it really tests anything.